cirq-core/cirq/circuits/frozen_circuit.py

```diff
@@ -25,13 +25,17 @@
     Union,
 )
 
-import numpy as np
-
-from cirq import devices, ops, protocols
 from cirq.circuits import AbstractCircuit, Alignment, Circuit
 from cirq.circuits.insert_strategy import InsertStrategy
 from cirq.type_workarounds import NotImplementedType
 
+import numpy as np
+
+from cirq import _import, _compat, devices, ops, protocols
+
+circuit = _import.LazyLoader("circuit", globals(), "cirq.circuits.circuit")
```

Comment:
What changed that we need this?  This looks scary.

Is this just for the _block_overlapping_deps?  If so (and we decide to keep this), we should mark a comment to remove it once that's no longer needed.    This may be a case of inappropriate sharing though and maybe we should just copy that function over, since it is small, temporary, and lint will pick up once it is no longer needed.

Reply:
Removed.